src/spaces/backing-services.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "../layouts/govuk.njk" %}
+
+{% block page_title %}
+  {{ space.entity.name }} - Overview
+{% endblock %}
+
+{% block main %}
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">Space</span>
+    {{ space.entity.name }}
+  </h1>
+
+  {% include "./_tabs.njk" %}
+
+  {% if space.entity.services.length > 0 %}
+
+    <h2 class="govuk-heading-m paas-space-between-heading-and-table">
+      This space contains {{ space.entity.services | length }} backing services
+    </h2>
+
+    <table class="govuk-c-table space-overview">
+      <thead class="govuk-c-table__head">
+        <tr class="govuk-c-table__row">
+          <th class="govuk-c-table__header name" scope="col">Service name</th>
+          <th class="govuk-c-table__header" scope="col">Type</th>
+          <th class="govuk-c-table__header" scope="col">Plan</th>
+          <th class="govuk-c-table__header" scope="col">State</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-c-table__body">
+        {% for service in space.entity.services %}
+          <tr class="govuk-c-table__row">
+            <th class="govuk-c-table__header" scope="row">
+              <a href="{{ linkTo('admin.organizations.spaces.services.view', {organizationGUID: organization.metadata.guid, spaceGUID: space.metadata.guid, serviceGUID: service.guid}) }}" class="govuk-link">{{ service.name }}</a>
```

Comment:
This as a link doesn't take you anywhere, should it?

Reply:
Yep it should ... good catch.

I have added a commit to fix it and added a test to catch duplicated names used for routes (which was the cause of the issue)